inputs/shock_tube3d_sph.ini

```diff
@@ -46,6 +41,4 @@ u0 = 0
 u1 = 0
 P0 = 1
 P1 = 0.1
-fx0 = 1
-fx1 = 0
```

Comment:
Please also remove the `fx0` and `fx1` in the `ParamSetup` of the shock tube setup